### DISCUSSION TITLE
光猫能正常访问Cloudflare的网站，路由器却会时连时断

### DISCUSSION THREAD

**Discussion by Killio07** [XIU2/CloudflareSpeedTest#301] [讨论]
光猫能正常访问Cloudflare的网站，路由器却会时连时断，路由器也没有特别设置什么东西，不知道为什么会这样，路由器大致表现为正常访问5分钟，然后就访问不了了，再过5-10分钟又正常了，正常时延迟和下载测试都是没有问题的，访问不了时测速基本都是75%丢包，下载速度0，请教一下各位大佬是什么原因

**Comment by pencilnav** [comment]
遇到就重新拨号，马上正常

**Comment by Killio07** [comment]
> 遇到就重新拨号，马上正常

可是光猫发出的wifi是正常的，光猫这边本身没有什么问题，不知道是路由器哪里设置上的问题

**Comment by pencilnav** [comment]
这个问题，应该去[Discussions](https://github.com/XIU2/CloudflareSpeedTest/discussions)提问吧...
